packages/cardano-graphql-db-sync/src/cardanoGraphqlDbSyncProvider.ts

```diff
@@ -42,6 +42,170 @@ export const cardanoGraphqlDbSyncProvider = (uri: string): CardanoProvider => {
     return CardanoGraphqlToOgmios.tip(response.cardano.tip);
   };
 
+  const networkInfo: CardanoProvider['networkInfo'] = async () => {
+    const query = gql`
+      query {
+        activeStake_aggregate {
+          aggregate {
+            sum {
+              amount
+            }
+          }
+        }
+        ada {
+          supply {
+            circulating
+            max
+            total
+          }
+        }
+        cardano {
+          currentEpoch {
+            lastBlockTime
+            number
+            startedAt
+          }
+        }
+      }
+    `;
+
+    type Response = {
```

Comment:
I'm wondering if there's a way to generate response type from graphql schema+query

Reply:
We can use https://www.graphql-code-generator.com/, and it's probably at the point where we can consider introducing it.